Seminar1/Task2.py

```diff
@@ -0,0 +1,12 @@
+# Задача 2: Найдите сумму цифр трехзначного числа.
+# *Пример:*
+# 123 -> 6 (1 + 2 + 3)
+# 100 -> 1 (1 + 0 + 0) |
+
+a = int(input('Введите трехзначное, целое число: '))
+sum = 0
+while a > 0:
+    sum = int(sum + a % 10)
```

Comment:
Во-первых, тут совсем не обязательно использовать функцию `int`, так как остаток отделения будет целым. Можно использовать быструю запись
```suggestion
    sum += a % 10
```
А еще, довольно плохая практика использовать в качестве имен переменных имена уже существующих встроенных инструментов Python. Я имею, имею в виду переменную `sum`, так как существует такая функция.

Reply:
Да, я вкурсе, но в данном случае на это просто не обратил внимание, учту